packages/i18n/src/locales/en/options/bella.yml

```diff
@@ -10,6 +10,10 @@ bustSpanEase:
   title: Bust span ease
   description: Controls the amount of (horizontal) ease added to your bust span when locating the bust point.
 
+shoulderToShoulderEase:
+  title: Shoulder to Shoulder ease
+  description: Controls the amount of ease between you shoulders. Initially set to -.5% because Bella implements a block that is used in the industry.
```

Comment:
```suggestion
  description: Controls the amount of ease between your shoulders. Initially set to -.5% because Bella implements a block that is used in the industry.
```